<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>454</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>348</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>348</width>
    <height>454</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe Script</family>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Menu Manager Pro</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Menu ManagerPro/menu.ico</normaloff>../Menu ManagerPro/menu.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* Estilo para los frames */
QFrame {
    background-color: #fefefe; /* color blanco viejo */
    border: 2px solid black;
    border-radius: 5px;
}


/* Estilo para los labels de usuario y contraseña */
QLabel {
    background-color: transparent;
    color: #003366; /* azul oscuro */
    font-size: 20px;
    font: 75 16pt &quot;Segoe Script&quot;;

}

/* Estilo para los campos de entrada (QLineEdit) */
QLineEdit {
    border: none;
    border-bottom: 2px solid black;
    border-radius: 5px;
    background-color: transparent;
    padding: 5px;
}

QLineEdit:focus {
    border-bottom: 2px solid #003366; /* borde azul al seleccionar */
}

/* Estilo para el botón de submit */
QPushButton {
    background-color: #003366; /* azul oscuro */
    color: white;
    border-radius: 5px;
    padding: 8px 15px;
    font-size: 14px;
    border: none;
}

QPushButton:hover {
    background-color: #005b99; /* color más claro al pasar el ratón */
    font-size: 15px; /* agranda un poco las letras */
}
</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhSensitiveData</set>
  </property>
  <widget class="QWidget" name="loginFrame">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_error_login">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>340</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 6pt &quot;Segoe Script&quot;;
color: rgb(170, 0, 0);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_login_login">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Script</family>
       <pointsize>48</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Estilo para el label principal */
QLabel {
    background-color: transparent;
    
	color: rgb(0, 0, 0);
    font-size: 48px;
    font-weight: bold;
    font: 75 48pt &quot;Segoe Script&quot;;
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="entry_pass_login">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="entry_user_login">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_user_login">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>User</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>429</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(24, 24, 24);
font: 87 8pt &quot;Segoe UI Black&quot;;
border: none;
</string>
     </property>
     <property name="text">
      <string>© Leandro Borsini</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_pass_login">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_submit_login">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>370</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
